### PR TITLE
Updated webapck config for css bundling

### PR DESCRIPTION
**Pull request by anishsarangi**
@vbanos I have bundled css files required for *feedback glyphicon* such that the user dont have to use npm install. While learning Webpack, I realise it is better to create a separate folder for bundled file. And I have updated the new build file's link in all the html files.
Thank you.

### CODE REVIEW

**Comment by anishsarangi** [comment]
@vbanos Please check the latest commits and changes. I have deleted the dist folder and bundled font-awesome and bootstrap. 
Thank you.

File: scripts/popup.js [hunk 1]
```diff
@@ -172,6 +172,7 @@ function display_suggestions(e) {
   }
 }
 function open_feedback_page() {
+  console.log("hi");
```
Comment (vbanos) [inline scripts/popup.js]: Please remove unnecessary `console.log`

**Comment by vbanos** [comment]
@anishsarangi we don't need any more `/css/font-awesome.min.css` and `/css/bootstrap.min.css` right?
Please remove them.

Everything else looks fine.

**Comment by anishsarangi** [comment]
Thank you @vbanos . I have updated the PR

**Pull request merged**